activerecord/lib/active_record/nested_attributes.rb

```diff
@@ -307,7 +307,7 @@ module ClassMethods
       # [:allow_destroy]
       #   If true, destroys any members from the attributes hash with a
       #   <tt>_destroy</tt> key and a value that evaluates to +true+
-      #   (e.g. 1, '1', true, or 'true'). This option is off by default.
+      #   (e.g. 1, '1', true, or 'true'). This option is +false+ by default.
```

Comment:
This could be a separate PR.

Reply:
Good point! I've opened https://github.com/rails/rails/pull/49936.